frontend/components/modal/ModalCommandPalette.vue

```diff
@@ -0,0 +1,196 @@
+            <Icon
+              name="octicon:comment-discussion"
+              size="1em"
+              :alt="$t('_global.discussions')"
+            />
+          </div>
+          <div class="group command-palette-element-hover">
+            {{ $t("_global.discussions") }}
+            <span class="jump-link">{{
+              $t("components.modal-command-palette.jump-to")
+            }}</span>
+          </div>
+        </div>
+      </router-link>
+
+      <DialogTitle class="font-display flex justify-between mt-3">
+        <p class="text-3xl md:responsive-h2 font-bold">
+          {{ $t("_global.organization") }}
+        </p>
+      </DialogTitle>
+      <router-link to="/organizations">
+        <div class="flex mt-2">
+          <div>
+            <Icon
+              name="codicon:organization"
+              size="1em"
+              :alt="$t('components.modal-command-palette.organization-stub')"
+            />
+          </div>
+          <div class="group command-palette-element-hover">
+            {{ $t("components.modal-command-palette.organization-stub") }}
+            <span class="jump-link">{{
+              $t("components.modal-command-palette.jump-to")
+            }}</span>
+          </div>
+        </div>
+      </router-link>
+      <router-link to="/organizations">
+        <div class="flex mt-2">
+          <div>
+            <Icon
+              name="codicon:organization"
+              size="1em"
+              :alt="$t('components.modal-command-palette.organization-stub')"
+            />
+          </div>
+          <div class="group command-palette-element-hover">
+            {{ $t("components.modal-command-palette.organization-stub") }}
+            <span class="jump-link">{{
+              $t("components.modal-command-palette.jump-to")
+            }}</span>
+          </div>
+        </div>
+      </router-link>
+      <router-link to="/organizations">
+        <div class="flex mt-2">
+          <div>
+            <Icon
+              name="codicon:organization"
+              size="1em"
+              :alt="$t('components.modal-command-palette.organization-stub')"
+            />
+          </div>
+          <div class="group command-palette-element-hover">
+            {{ $t("components.modal-command-palette.organization-stub") }}
+            <span class="jump-link">{{
+              $t("components.modal-command-palette.jump-to")
+            }}</span>
+          </div>
+        </div>
+      </router-link>
+    </template>
+  </ModalBase>
+</template>
+
+<script setup>
+import { useMagicKeys, whenever } from "@vueuse/core";
+
+// TODO: Maybe there is a better / more future-proof solution than userAgent?
+const isMacOS = /Mac/.test(navigator.userAgent);
+
+const { slash, meta_k, ctrl_k } = useMagicKeys({
+  passive: false,
+  onEventFired(e) {
+    if (["slash", "meta_k", "ctrl_k"].includes(e.key) && e.type === "keydown")
+      e.preventDefault();
+  },
+});
+
+whenever(slash, () => {
```

Comment:
Quick thing that I'm realizing here @mattburnett-repo is that I didn't do a good job of explaining that slash should maintain its old functionality for this :)

Ultimately activist will have two kinds of search: the slash one that's from the search bar that searches within the current domain for events or organizations or anything if we're not within a sub domain, and then the command palette that should only be triggered by `meta_k` or `ctrl_k`.

Would it be possible to:
- Remove the slash functionality from this
- Uncomment those lines in the search bar to show the command palette shortcut indicators

I'll then go through and do some styling. After that we should be good!

Reply:
No worries @andrewtavis . We had earlier some discussion about overall CommandPalette functionality and I probably got confused during that discussion. 

I've implemented your suggestions and they will appear in the next PR.

Reply:
Looking forward to checking out the finalized version later, @mattburnett-repo! Thanks for the quick reaction :) :)